src/ServerConfigParams.ts

```diff
@@ -70,79 +79,54 @@ interface ServerDependencies {
   txStoreManager: TxStoreManager;
 }
 
-const ERROR_DISABLE_SPONSOR_TX_NOT_CONFIGURED =
-  'The param disableSponsoredTx should be properly configured. Valid values are true or false.';
 const ERROR_GAS_FEE_PERCENTAGE_NEGATIVE =
```

Comment:
it should not be better to have the errors on a different file?

Reply:
Changed.